stix_shifter_modules/vectra/stix_translation/json/to_stix_map.json

```diff
@@ -103,16 +104,18 @@
     }
   ],
   "certainty": [
-    {
-      "key": "x-ibm-finding.confidence",
-      "object": "detection"
-    },
```

Comment:
this mapping looks valid. not sure why you need to remove this and put it in `certainty_score` field?

Reply:
Previously, the certainty field was mapped to both x-ibm-finding:confidence and x-ibm-ttp-tagging:confidence fields as given below: 
"certainty": [
    {
      "key": "x-ibm-finding.confidence",
      "object": "detection"
    },
    {
      "key": "x-ibm-ttp-tagging.confidence",
      "object": "ttp_finding",
      "transformer": "ConvertToReal"
    }
  ],

As per the schema definition x-ibm-ttp-tagging:confidence value is in range 0-1 and x-ibm-finding:confidence value is in the range 1-100.
From the Vectra Datasource, certainty value is coming in the range 0-100. Hence x-ibm-ttp-tagging:confidence requires a transformer to convert certainty value from the range [0-100] to [0-1].
Since the same field certainty was mapped to two different STIX fields x-ibm-ttp-tagging:confidence and x-ibm-finding:confidence, when the transformer “ConvertToReal“ was applied to x-ibm-ttp-tagging:confidence it was applied to both the fields x-ibm-ttp-tagging:confidence and x-ibm-finding:confidence when tested from CP4S.
Hence to work around this issue from CP4S, a copy of certainty field was made as certainty_score. Field certainty_score was mapped to x-ibm-finding:confidence and the issue was resolved .

Reply:
As per your suggestion removed the code